lua/wire/client/texteditor.lua

```diff
@@ -3109,6 +3109,7 @@ do -- E2 Syntax highlighting
 		["@inputs"] = 1, -- directive yellow, types orange, rest normal
 		["@outputs"] = 1,
 		["@persist"] = 1,
+		["@rawinputs"] = 1,
```

Comment:
I have no idea what this does, since you didn't give a description.
I don't know if you have implemented all/none yet or not